kbase-extension/static/kbase/js/widgets/function_output/kbaseRESKESearchResultDemo.js

```diff
@@ -0,0 +1,535 @@
+        srvVersion:"",
+        genericClient: null,
+
+        PAGE_SIZE: 20,
+        searchText: "",
+        searchObjectType: "",
+        $container: null,
+        $searchTypesResultPane: null,
+        $searchObjectsResultPane: null,
+
+        listTypesInput:{
+            object_type:"data_type, but it is optional"
+        },
+
+
+        escapeSymbols: {
+            '&': '&amp;',
+            '<': '&lt;',
+            '>': '&gt;',
+            '"': '&quot;',
+            "'": '&#39;',
+            '/': '&#x2F;',
+            '`': '&#x60;',
+            '=': '&#x3D;',
+            '\n': ' ',
+            '\r': ''
+        },
+
+        // examples...
+        listTypesOutput:{
+            "types": {
+                "Genome":{
+                    type_name: "Genome",
+                    type_ui_title: "Genome",
+                    keys: [{key_name:"key1", key_ui_title: "title1", key_value_type:"value_type"}],
+                },
+                "Feature":{
+                    type_name: "Feature",
+                    type_ui_title: "Feature",
+                    keys: [{key_name:"key1", key_ui_title: "title1", key_value_type:"value_type"}],
+                },
+                "Domain":{
+                    type_name: "Domain",
+                    type_ui_title: "Domain",
+                    keys: [{key_name:"key1", key_ui_title: "title1", key_value_type:"value_type"}],
+                }
+            }
+        },
+        searchTypesInput:{
+            match_filter:{
+                full_text_in_all: "text",
+                access_group_id: 123,
+                object_name: "name",
+                parent_guid: 234234,
+                timestamp: {},
+                lookupInKeys: {}
+            },
+            access_filter:{}
+        },
+
+        searchTypesOutput:{
+            type_to_count: {
+                "Genome": 140,
+                "Feature": 120,
+                "Domain": 10,
+            },
+            search_time: 123
+        },
+
+        searchObjectsInput:{
+            object_type: "object type",
+            match_filter: {
+                full_text_in_all: "text",
+                access_group_id: 123,
+                object_name: "name",
+                parent_guid: 234234,
+                timestamp: {},
+                lookupInKeys: {}
+            },
+            sorting_rules: [],
+            access_filter: {},
+            pagination: {
+                start: 0,
+                count: 0    
+            },
+            post_processing:{
+                ids_only: false,
+                skip_info: false,
+                skip_keys: false,
+                skip_data: false,
+                data_includes: [] 
+            }
+        },
+        searchObjectsOutput: {
+            pagination :{
+                start: 21,
+                count: 40        
+            },
+            sorting_rules: [],
+            objects: [
```

Comment:
A lot of this stuff seems hard-coded. Is this just for the demo for now? Should it move to a config file?